Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj

```diff
@@ -60,4 +54,8 @@
     <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="2.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Models\" />
```

Comment:
This folder can be removed from the project if it is not being used

Reply:
Fixed it . Thanks